src/features/projectsV2/ProjectDetails/components/ProjectInfo.tsx

```diff
@@ -16,7 +16,7 @@ import ImageSlider from './microComponents/ImageSlider';
 import { ExtendedProject } from '../../../common/types/projectv2';
 
 interface ProjectInfoProps {
-  project: ExtendedProject;
+  project: ExtendedProject | null;
   isMobile: boolean;
```

Comment:
I don't think `null` should be accepted as a possible value for project. It allows us to skip the check while using `ProjectInfo` which is not advisable.